### ISSUE TITLE
Add SaaS reference(s) to the 'Usage type' in our CRM post terminology stds discussion

### ISSUE THREAD

**Issue by jon-mc-git** [bcgov/cloud-pathfinder#1654]
**Describe the issue**
New field request for CRM

Need way to classify incoming projects that involve SaaS only, internal Platform-SaaS or external SaaS provider - Need categories & terms standardization.

Label clearly in the CRM.  Regardless of which team SaaS or CPF deals with it, we use our CRM for tracking conversations we're having within BC Gov, so it's helpful for us to know where that SaaS request ended up (so we can view it filtered easily)

**Additional context**
More projects will be coming in with these types of patterns so a naming convention should be discussed for the types of SaaS that we'll be dealing with and any nuances we may discover
- basically asking for extra labels in a field

**Definition of done**
- Initial discussion help on stds for SaaS Cloud categories and naming convention moving forward
- Final decision on draft terminology for usage type (Data Services, Infrastructure, SaaS, etc)
- Input of those new types into current CRM instance
